### DISCUSSION TITLE
We are having a problem billing your account. Please update your payment method or call your payment provider for details on why the transaction failed. You can contact support with any questions

### DISCUSSION THREAD

**Discussion by viwanoudenis** [community/community#140089] [Copilot Conversations]
help , i have 10$ in my account to start using git copilot , but i have this problem with invalid payment method

**Comment by davevad93** [comment]
Hi @viwanoudenis ,

That means there's a problem with your [billing settings](https://github.com/settings/billing/payment_information). You should contact support by opening a ticket [here](https://support.github.com/contact/account), select `Billing, payments or receipts` and ask for assistance.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @viwanoudenis ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
